ffv1.md

```diff
@@ -933,8 +953,8 @@ pseudo-code                                                   | type
 ConfigurationRecord( NumBytes ) {                             |
     ConfigurationRecordIsPresent = 1                          |
     Parameters( )                                             |
-    while( remaining_bits_in_bitstream( NumBytes ) > 32 )     |
-        reserved_for_future_use                               | u(1)
+    while( remaining_symbols_in_syntax )                      |
```

Comment:
should be a function (with parenthesis) and with a description as remaining_bits_in_bitstream() is.